network.go

```diff
@@ -230,6 +231,21 @@ func (s *sandbox) removeInterface(netHandle *netlink.Handle, iface *types.Interf
 	return nil, nil
 }
 
+func expectLinkUpdate(ch <-chan netlink.LinkUpdate, ifaceName string) bool {
+	for {
+		timeout := time.After(time.Second * 5)
```

Comment:
You could define this outside of the `for` loop. Also, assign a constant for the 5 seconds, that would be something like `LinkUpdateTimeout` for instance.
```go
...
const LinkUpdateTimeout = 5
...
func expectLinkUpdate(ch <-chan netlink.LinkUpdate, ifaceName string) bool {
        timeout := time.After(time.Second * LinkUpdateTimeout)
        for {
....
```